lib/syskit/process_managers/remote/process.rb

```diff
@@ -73,7 +73,10 @@ def resolve_all_tasks
                         mapped_name = mapped_name_of(deployed_task.name)
                         unless (ior = @ior_mappings[mapped_name])
                             raise IORNotRegisteredError,
-                                  "no IOR is registered for #{task_name}"
+                                  "no IOR is registered for #{deployed_task.name}. The "\
+                                  "task deployment name on the process server differs "\
+                                  "to the task deployment name in syskit, "\
+                                  "#{deployed_task.name}"
```

Comment:
The second part of the message is only a hint

The more general hint is "the deployment model used by the process server may differ from the one used by the syskit main instance"